File: app/src/main/java/com/crevion/apps/cleanandroidcode/networking/NetworkService.java
package com.crevion.apps.cleanandroidcode.networking;

import com.crevion.apps.cleanandroidcode.models.CityListResponse;


import retrofit2.http.GET;
import rx.Observable;

/**
 * Created by yusufaw on 1/27/18.
 */

public interface NetworkService {
    @GET("v1/city")
    Observable<CityListResponse> getCityList();
}

File: app/src/main/java/com/crevion/apps/cleanandroidcode/networking/NetworkService.kt
package com.crevion.apps.cleanandroidcode.networking

import com.crevion.apps.cleanandroidcode.models.CityListResponse
import retrofit2.http.GET
import rx.Observable

/**
 * Created by yusufaw on 2/9/18.
 */

interface NetworkService {
    @GET("v1/cities")
    fun getCityList(): Observable<CityListResponse>
}